src/base.jl

```diff
@@ -2266,6 +2266,7 @@ function IS.compare_values(
     y::T;
     compare_uuids = false,
     exclude = Set{Symbol}(),
+    match_fn = IS.isequivalent,
```

Comment:
This function probably needs a version that takes less arguments and forwards to the version with the `match_fn`

Reply:
Done